### ISSUE TITLE
Unecessary dependency on preview version of System.ComponentModel.Annotations.

### ISSUE THREAD

**Issue by rdecastroLGC** [elsa-workflows/elsa-core#3018]
Hello guys,

While developing in a project that needed a _.net SDK_ bump to version `6.0` we've come across a reference to `System.ComponentModel.Annotations Version=6.0.0-preview.4.21253.7` in the `Elsa.Abstractions` project.
From what I've seen there's no need to use the preview version and the solution builds and runs with the version 5.0.0.
Don't you agree that we shouldn't take a dependency on a preview version of a library?

Keep up the good work,
Cheers!

**Comment by rdecastroLGC** [comment]
Hello guys! Any opinion on this?

**Comment by rdecastroLGC** [comment]
Thank you, @sfmskywalker! 
Cheers 👍

**Comment by rdecastroLGC** [comment]
@sfmskywalker, actually we noticed you completely removed the reference to the package but the package is needed. What we've tested over here was replacing the ref. you've deleted with `<PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />` in the **Elsa.Abstractions** _csproj_.

**Comment by sfmskywalker** [comment]
I was wondering about that. I tried a build, but should have done a full rebuild and I would have noticed the same. Thanks for letting me know, will fix 👍🏻

**Comment by rynkadink** [comment]
> It looks like only two projects in the Elsa solution are depending on this package, so I'm kinda wondering if I shouldn't leave out the System.ComponentModel.Annotations package from Elsa.Abstractions - since that package doesn't depend on it. Others depend on it through transference only. Thoughts?

I went on and checked it out as well. From what I could see the two projects that need the library mentioned above (_System.ComponentModel.Annotations_) are `Elsa.Activities.File` and `Elsa.Activities.BlobStorage`. 

I agree with you guys, we should not rely on having transient dependencies - if it's not needed directly in the project's code then it's not doing anything there besides possibly bringing us headaches in the future.

--- 

I opened a pull-request #3076 with the changes that we need to make (if you don't want to make the changes yourself).
